### PR TITLE
Network update (sprint 42)

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: blog/2023-08-18-network.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+it with an `Any` type on the Haskell side and made all remaining tests & specs
+use it.  This simplified the specifications and made it easier to understand
+which parts are defined in the spec, and which parts are left unspecified.
+See [ouroboros-network#4595].
+
+### Ouroboros-Network-Framework API changes
+
+We also released `ouroboros-network-framework` and other network components.
+The `ouroboros-network-framework` package contains a redesign of API exposed to
+`ouroboros-cosnensus`.  We consolidated, cleaned it and made it easier to
```
Comment (njd42) [inline blog/2023-08-18-network.md:33]: spelling of 'consensus' in repository name

File: blog/2023-08-18-network.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+And provides a shell which contains all the build tools, including `ghc-9.6`,
+`hls`, `cddl`, and more.  See [ouroboros-network#4640],
+[ouroboros-network#4643].
+
+
+## Other contributions
+
+### Cardano Network Service Assurance
+
+* The busywork and writeup in finishing up the CNSA, first stage (first
```
Comment (njd42) [inline blog/2023-08-18-network.md:55]: do you really mean 'busywork' here - that is usually a term for work not needed to be done , I don't think you intended the derogatory meaning here! :)

**Review by njd42 — CHANGES_REQUESTED** [review]
Some spelling/grammar issues - contents seem basically fine

File: blog/2023-08-18-network.md (hunk 2 above)
Comment (coot) [reply blog/2023-08-18-network.md:55]: Oh, I didn't know, I'll remove it. I got this part from Mark, maybe it's UK / US difference?

**Review by bolt12 — APPROVED** [review]
LGTM!

**Pull request merged**